executable/te.ksy

```diff
@@ -0,0 +1,125 @@
+meta:
+  id: te
+  title: TE (Terse Executable) file format. A stripped version of PE (Portable Executable) format.
+  application: UEFI
+  file-extension:
+    - efi
+    - te
+  ks-version: 0.7
+  license: CC0-1.0
+  endian: le
+doc-ref: https://uefi.org/sites/default/files/resources/PI_Spec_1_6.pdf
+seq:
+  - id: te
+    size: 0x28
+    type: te_header
+  - id: sections
+    type: section
+    repeat: expr
+    repeat-expr: te.number_of_sections
+types:
+  te_header:
+    seq:
+      - id: signature
+        contents: "VZ"
+      - id: machine
+        type: u2
+        enum: machine_type
+      - id: number_of_sections
```

Comment:
Please consider following [style guide](https://doc.kaitai.io/ksy_style_guide.html) here and using something like `num_sections`?